src/other/broadcasting.jl

```diff
@@ -139,7 +137,8 @@ if isdefined(Base, :dotgetproperty)
             # TODO: double check that this is tested
             return LazyNewColDataFrame(df, Symbol(col))
         else
-            Base.depwarn("In the future this operation will allocate a new column " *
+            # TODO: remove the deprecation in DataFrames.jl 1.4 release
+            Base.depwarn("In the 1.4 release of DataFrames.jl this operation will allocate a new column " *
                          "instead of performing an in-place assignment.", :dotgetproperty)
```

Comment:
```suggestion
            Base.depwarn("In the 1.4 release of DataFrames.jl this operation will allocate a new column " *
                         "instead of performing an in-place assignment. Use `v = df.col; col .= ...` instead.",
                         :dotgetproperty)
```